### DISCUSSION TITLE
brightness control works but Volume doesnt work.

### DISCUSSION THREAD

**Discussion by tarunprakashsingh** [MonitorControl/MonitorControl#1149] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Volume control doesnt work.... volume up/down and mute shows up on the screen but volume is not effected.
Brightness control works perfectly. 
I am using Macbook pro Max. external monitor LG 32UN650-W.
 

### Expected behavior

volume should go up/down or mute the volume

### Environment Information (please complete the following information)

```markdown
- macOS version: monterey
- Mac model: macbook pro max
- MonitorControl version: 4.1.0 nuild 7034
- Monitor(s): lg 32UN650-W.
- Monitor Cable(s)/Connection(s): hdmi
- Apple Silicon/M1 (yes or no): yes
```

**Comment by brian-petersen** [comment]
Same issue here! Pretty much same setup except I'm using a DP cable.

**Comment by tarunprakashsingh** [comment]
so HDMI is not working?
Can you suggest which cable you are using, so I get the same one and
avoid issues.

On Tue, Aug 9, 2022 at 3:58 PM Brian Petersen ***@***.***>
wrote:

> Same issue here! Pretty much same setup except I'm using a DP cable.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/MonitorControl/MonitorControl/issues/1134#issuecomment-1209814586>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTGSKUJCJHMYANOMIITC3VYKZ5TANCNFSM56BQCRGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thanks
Tarun Singh

Like Us on Facebook CricketDepotOnline
<http://www.facebook.com/CricketDepotOnline>
Like Us on Facebook XTreme Cricket USA
<http://www.facebook.com/XtremeCricketUSA>

**Comment by tarunprakashsingh** [comment]
> Same issue here! Pretty much same setup except I'm using a DP cable.

@brian-petersen can you give the link for this cable so i can get the same. avoiding any other issues :)

**Comment by waydabber** [comment]
DDC control through the built-in HDMI pro does not work. Brightness control seemingly works since by default a combined software dimming and DDC control is applied (probably only the bottom 50% of the scale actually changes the brightness in this case). You should use USB-C or DisplayPort to connect the display.

@brian-petersen - DP should work, unless the display does not process DDC volume commands (many displays don't, actually I just happen to have a display like that as well). It's entirely firmware dependent and a hit-or-miss thing.
